x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,574 @@
+Create a new CA certificate and add it to your existing CA truststore. After
+{es} reloads the file, you can remove the old CA certificate from your
+truststore.
+
+NOTE: The following examples use PKCS#12 files, but the same steps apply to JKS
+keystores.
+
+. Open the `ES_PATH_CONF/elasticsearch.yml` file and check the names of the
+security certificates that are currently in use. You'll use the same names
+for your new certificates.
++
+In this example, the keystore and truststore are using different files.
+Your configuration might use the same file for both the keystore and the 
+truststore. Elastic recommends using separate files. 
++
+[source,yaml]
+----
+xpack.security.transport.ssl.keystore.path: config/elastic-certificates.p12
+xpack.security.transport.ssl.keystore.type: PKCS12
+xpack.security.transport.ssl.truststore.path: config/elastic-stack-ca.p12
+xpack.security.transport.ssl.truststore.type: PKCS12
+----
+
+. On *any* node in your cluster, generate a new CA certificate. You only need
+to complete this step one time.
++
+NOTE: If your organization has its own CA, then use that CA to generate a new
+CA certificate in PEM format.
++
+[source,shell]
+----
+./bin/elasticsearch-certutil ca --pem
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `--pem`:: Generates a directory containing a CA certificate and key in PEM
+   format instead of PKCS#12. 
+====
+
+    a. Enter a name for the compressed output file that will contain your
+    certificate and key, or accept the default name of `elastic-stack-ca.zip`.
+
+    b. Unzip the output file. The resulting directory contains a CA certificate
+    (`ca.crt`) and a private key (`ca.key`).
++
+--
+IMPORTANT: Keep these file in a secure location as they contain the private key
+for your CA. 
+--
+
+. On *every* node in your cluster, import the new `ca.crt` certificate into your
+existing CA truststore. This step ensures that your cluster trusts the new CA
+certificate. This example uses the Java `keytool` utility to import the
+certificate into the `elastic-stack-ca.p12` CA truststore.
++
+[source,shell]
+----
+keytool -importcert -trustcacerts -noprompt -keystore elastic-stack-ca.p12 \
+-storepass <password>  -alias new-ca -file ca.crt
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `-keystore`:: Name of the truststore that you are importing the new CA
+   certificate into. 
+
+   `-storepass`:: Password for the CA truststore. 
+
+   `-alias`:: Name that you want to assign to the new CA certificate.
+
+   `-file`:: Name of the new CA certificate to import.
+====
+
+. Check that the new CA certificate was added to your truststore. The output
+should contain both the existing CA certificate and your new certificate. Each
+CA certificate is identified by the `friendlyName` attribute in the output.
++
+When prompted, enter the password for the CA truststore.
++
+[source,shell]
+----
+openssl pkcs12 -info -in config/elastic-stack-ca.p12
```

Comment:
lets stick to one external tool when possible, maybe: `keytool -keystore config/elastic-stack-ca.p12 -list` ?